### PR TITLE
Bumping version to 2.0.2

### PR DESCRIPTION
**Pull request by VachaShah**
Signed-off-by: Vacha Shah <vachshah@amazon.com>

### Description
Bumping version to 2.0.2

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by BrianHawley** [comment]
Can we wait until #76 is merged? That way we can do an opensearch-transport upgrade too.

**Comment by VachaShah** [comment]
> Can we wait until #76 is merged? That way we can do an opensearch-transport upgrade too.

The 2.0.2 is already released since its a patch release. We can add the Faraday support in 2.1 since it is a feature.

**Review by peterzhuamazon — APPROVED** [review]

**Review by dblock — APPROVED** [review]

**Pull request merged**
